internal/repotools/cmd/updatemodulemeta/main.go

```diff
@@ -175,7 +183,7 @@ func writeModuleMetadata(dir string, goPackage string, version string) (err erro
 
 	return metadataTemplate.Execute(f, metadata{
 		Package: goPackage,
-		Version: version,
+		Version: strings.TrimPrefix(version, "v"),
```

Comment:
Mainly since the SDK version constant has always been reported without it. Same with the product version that is sent in the user-agent, so opt-ed for consistency here.